### DISCUSSION TITLE
Can I send notifications to a phone (Android) using a watch app?

### DISCUSSION THREAD

**Discussion by kyw34243242** [zepp-health/zeppos-forum#386] [Q&A] [answered]
I want to make a notification with a specific message appear on my phone when pressing a button using the watch app.
I plan to use Samsung phone's Bixby Routines (to trigger routines based on specific notifications) for various interactions (mainly IoT or phone control).
I would appreciate it if you could provide guidance on how to proceed or share links and resources on what I need to learn.

**Comment by silver-zepp** [comment, marked as answer]
Look into this example [webserver-data-extraction](https://github.com/silver-zepp/zeppos-autogui/tree/master/other-autogui-examples/webserver-data-extraction). 

For the most part, you want to run a server on your phone (or some automation like Tasker that will do that for you), catch a message and show it as a notification.

**Reply by kyw34243242** [reply]
Thank you for the response:D  I made it possible to interact between the mini-program on Zepp OS on my phone and the watch buttons using the to-do list. I’m planning to integrate it with Tasker, but could you let me know the port for the HTTP request? Or could you tell me where I can modify the port?

**Reply by silver-zepp** [reply]
Awesome! Check out [this guide](https://youtu.be/0R9Go6tJqKY). I believe the port should be `1821` but you should be able to change it.

**Reply by kyw34243242** [reply]
Even if I send an HTTPS request to port 1821, Tasker does not detect it unless it’s to a standard online domain. As a workaround, I set it up to send the request to the Pushover site when a button is pressed, which then triggers an alarm on the phone via Pushover. Based on this alarm, it controls the IoT. It seems that monitoring all HTTPS requests on the device is impossible without creating an app that uses VPN. It would be ideal to control it via Bluetooth signals, but it doesn't seem to support this—am I correct? Anyway, thank you for your help.
